demo/k8suniquelabels_template.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: templates.gatekeeper.sh/v1alpha1
+kind: ConstraintTemplate
+metadata:
+  name: k8suniquelabels
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sUniqueLabel
+        listKind: K8sUniqueLabelList
+        plural: k8suniquelabels
+        singular: k8suniquelabel
```

Comment:
I noticed you named both`kind` and `singular`: `K8sRequiredLabels` here:
https://github.com/open-policy-agent/gatekeeper/blob/e841d24d7284f8cb0934fdb8fb725f3daba02481/demo/k8srequiredlabels_template.yaml#L9-L12

so should we update `kind` and `singular` here as `k8suniquelabels` or is the other demo incorrect?

Reply:
In this case the plural/singular refers to the number of labels.

`UniqueLabel` can only specify label singular.

`RequiredLabels` can require multiple labels at once.